todoapp/todos/views.py

```
@@ -1,25 +1,54 @@
+
+    success response for list
+    [
+      {
+        "name": "",
+        "done": true/false,
+        "date_created": ""
+      }
+    ]
     """
 
+    permission_classes = [rest_permissions.AllowAny]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)